src/pending.js

```diff
@@ -0,0 +1,144 @@
+var _ = require('lodash');
+var debug = require('debug')('stats-collector:pending');
+var taskcluster = require('taskcluster-client');
+
+const FLUSH_INTERVAL = 3; // seconds
```

Comment:
Statsum only reports every 5min anyways... So there is no need to report every 3s.